tests/test_fetch_resource.py

```diff
@@ -0,0 +1,87 @@
+
+import pytest
+from pathlib import Path
+import os
+
+from neuromaps_nhp.resources.fetch_resource import fetch_atlas, fetch_transform, fetch_resource, search_resources
+from neuromaps_nhp.config import config
+
+def test_fetch_atlas():
+    # Test fetching a known atlas resource
+    atlas_path = fetch_atlas(
+        resource_name="sphere",
+        source="Yerkes19",
+        density="32k",
+        hemisphere="left"
+    )
+    assert atlas_path is not None
+    assert os.path.exists(atlas_path)
```

Comment:
Looking at this test, and then going back to the the methods, we should just return a `Path` instead of a str. From there, can just call something like `atlas_path.exists()`

Further, for this test, I would probably just test that the path exists, which would imply that the path is not None